pkgs/development/interpreters/supercollider/default.nix

```diff
@@ -1,6 +1,8 @@
 { stdenv, fetchurl, cmake, pkgconfig
 , libjack2, libsndfile, fftw, curl, gcc
 , libXt, qtbase, qttools, qtwebkit, readline
+, withAlsa ? !stdenv.isDarwin, alsaLib             # ALSA is required for MIDI to work in SC
```

Comment:
There is no need for `withAlsa`: enable it for Linux unconditionally.

Reply:
I put in ALSA to be optional since it's indicated as optional in supercollider readme.

Reply:
Nevertheless, there is no use in making it optional in Nixpkgs.